compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -1711,12 +1719,12 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
         let output = if let Some((ret_id, span)) = make_ret_async {
             match kind {
                 FnDeclKind::Trait => {
-                    if !kind.impl_trait_in_trait_allowed(self.tcx) {
```

Comment:
The other usage of `tcx.features().return_position_impl_trait_in_trait` can probably just be replaced similarly, right?

Reply:
Not sure what you mean; do you mean combining `impl_trait_return_allowed` and `impl_trait_in_trait_allowed`? I think that would work out if that's what you meant

Reply:
Whoops, copypasta. I totally got my words backwards. I mean the other usage of `impl_trait_in_trait_allowed`, which can just be inlined to `tcx.features().return_position_impl_trait_in_trait`. Sorry!